### ISSUE TITLE
bug: A user with a member role can execute, and even delete other members' workspaces.

### ISSUE THREAD

**Issue by matifali** [coder/coder#5379]
# Context:
A user named `atif` who was initially the `owner` and the first user is set as a member only after transferring the `owner` role to a new user named `admin.`

# Expected:
User `atif` should not be able to start, stop, delete or interact with other users' workspaces. 

# Actual: 
User `atif` can start and even delete another user's workspace.

**Comment by Emyrk** [comment]
I believe this is because the first user is also set to the [org admin](https://github.com/coder/coder/blob/663f7a3f127a2461dfd4e707d3ebac33a8a47a36/coderd/rbac/builtin.go#L18-L18). Since the product only really uses 1 org, we don't really expose these roles atm.

Org admins can update/stop/delete workspaces in their organization.

**Comment by johnstcn** [comment]
To fix this issue for now, you can manually remove the `org-admin` role from the user `atif` with the following steps:

1) Grab a Coder session token from your active Coder browser window: Open Development Tools -> Application -> Cookies -> `coder_session_token` -> copy value.

**Note:** you will need to do this as a user with owner permissions _other_ than the user from whom you wish to remove these roles.

2) Determine the user ID of the user you wish to demote, as well as the organization ID.
```
curl -s https://<CODER_URL>/api/v2/users?q=<USERNAME>&limit=1 -H 'Coder-Session-Token: <TOKEN>'
```
This will emit data similar to the following:
```
{
  "users": [
    {
      "id": "6529e773-8c9c-46a8-ac2a-3a6c44423531", # this is the user ID
      "username": "USERNAME",
      "email": "USERNAME@DOMAIN.TLD",
      "created_at": "2022-12-07T16:32:35.494966Z",
      "last_seen_at": "2022-12-14T14:13:38.796237Z",
      "status": "active",
      "organization_ids": [
        "80d9228f-00aa-4436-9382-4aeda937db16" # this is the org ID
      ],
      "roles": [{"name":"owner","display_name":"Owner"}], 
      "avatar_url": ""
    }
  ],
  "count": 1
}
```
Note the values of `users[0].id` and `users[0].organization_ids[0]`. These are the user ID and organization id, respectively. Your values will almost certainly be different.

3) Run the following curl command to remove all organization roles from the user identified in the previous step
```
curl -s https://<CODER_URL>/api/v2/organizations/<ORG_ID>/members/<USER_ID>/roles -H 'Coder-Session-Token: <TOKEN>' -XPUT --data-raw '{"roles": []}'
```

Result: That user should no longer be able to view / edit / delete workspaces of other users.

**To revert this change:** simply re-run the above curl command with the role `organization-admin:<ORG_ID>`, e.g.

```
curl -s https://<CODER_URL>/api/v2/organizations/<ORG_ID>/members/<USER_ID>/roles -H 'Coder-Session-Token: <TOKEN>' -XPUT --data-raw '{"roles": ["organization-admin:<ORG_ID>"]}'
```

**Comment by matifali** [comment]
Thanks for looking into it. Due to security concerns you may delete this issue if you wish so.

**Comment by Emyrk** [comment]
> Thanks for looking into it. Due to security concerns you may delete this issue if you wish so.

I do not think it is a security concern. In the PR I merged I demoted all `org-admins` -> `org-members`. So we rely on the site wide permissions for everything.

This is mainly due to us only using 1 organization at the moment, but the back-end supports multiple. So these roles are essentially "hidden" from the UI.

Until we support Organizations in the UI, the merged PR just keeps everyone at just an "organization-member". 

TBD on if we should even allow api changes to this atm. :thinking:
